app/src/cli/main.ts

```diff
@@ -0,0 +1,18 @@
+import * as ChildProcess from 'child_process'
+import * as Path from 'path'
+
+const args = process.argv.slice(2)
+
+// At some point we may have other command line options, but for now we assume
+// the first arg is the path to open.
+const pathArg = args.length > 0 ? args[0] : ''
+const repositoryPath = Path.resolve(process.cwd(), pathArg)
```

Comment:
>Related: what should happen if we pass in an invalid path here?

So it currently still opens the "Add Local Repository" with an invalid path. I _think_ that's right.